docs/user/datastores/index_config.rst

```diff
@@ -604,19 +604,48 @@ the trait ``org.locationtech.geomesa.index.planning.QueryInterceptor``:
         * @param query query
         */
       def rewrite(query: Query): Unit
+
+      /**
+       * Hook to allow interception of a query after extracting the query values
+       *
+       * @param strategy query strategy
+       * @return an exception if the query should be stopped
+       */
+      def guard(strategy: QueryStrategy): Option[IllegalArgumentException] = None
     }
 
 Interceptors must have a default, no-arg constructor. The interceptor lifecycle consists of:
 
 1. The instance is instantiated via reflection, using its default constructor
 #. The instance is initialized via the ``init`` method, passing in the data store containing the simple feature type
-#. ``rewrite`` is called repeatedly
-#. The instance is cleaned up via the ``close`` method
+#. ``rewrite`` and ``guard`` are called once for each query
+#. When the data store is disposed, the instance is cleaned up via the ``close`` method
 
 Interceptors will be invoked in the order they are declared in the user data. In order to see detailed information
 on the results of query interceptors, you can enable ``TRACE``-level logging on the class
 ``org.locationtech.geomesa.index.planning.QueryRunner$``.
 
+Temporal Query Guard
+++++++++++++++++++++
+
+GeoMesa provides a basic implementation of a query interceptor that will block overly broad queries
+(which can overwhelm the system). A maximum temporal duration must be specified, and any query which attempts
+to return a larger time period will be stopped. This will not affect queries against indices that do not have
+a temporal component (for example, feature ID and attribute queries).
+
+To enable the guard, set ``geomesa.query.interceptors`` to
```

Comment:
This setting is a comma-separated list of classes, right?  

Would it be more precise to state that one ought to be adding this to the list of interceptors?

(I'm being very precise to make sure I understand.  This may be adding complexity to the docs, so feel free to push back.)

Reply:
right, i will clarify.